man/systemd.mount.xml

```diff
@@ -226,6 +226,21 @@
         for details.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>x-systemd.wanted-by=</option></term>
+        <term><option>x-systemd.required-by=</option></term>
+
+        <listitem><para>Configures a <varname>Wants=</varname>
+        dependency or <varname>Requires=</varname> between the specified
+        unit or target and the created mount unit.  This option may be
```

Comment:
this should say clearly which unit requires which other unit. right now it just says there's a dependency between them, but no so clearly which side requires which other side